src/components/new-request/name-check/quick-search-names.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-row id="quick-search-names" no-gutters>
+    <v-col class="pa-3">
+      <v-row class="quickSearchNames" v-for="name of namesList" :key="name.name" no-gutters>
+        <v-col>
+          <b v-if="name.type==='exact'">{{name.name}}</b>
+          <span v-else>{{name.name}}</span>
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import MainContainer from '@/components/new-request/main-container.vue'
+
+@Component({
+  components: { MainContainer }
+})
+export default class QuickSearchNames extends Vue {
+  // Global getter
+  @Prop() namesList: Array<{ name: string, type: string }>
```

Comment:
Seeing that you're expecting a specific value(s) for **type** (eg, "exact"), could you turn this into an enum?

Reply:
yeah I can haha. Also I will update that type to the interface

Reply:
Yes, I meant the type.

Well, the name, too, if it's a set of possible values :)

Reply:
ha, the name can be anything, but I will add it for the type. I meant the interface for that object (it's used in actions)

Reply:
Is this not fixable, or future?